wikibase/README.md

```diff
@@ -56,3 +57,10 @@ File                              | Description
 --------------------------------- | ------------------------------------------------------------------------------
 `/LocalSettings.php.template`     | Template for Mediawiki Localsettings.php (substituted to `/var/www/html/LocalSettings.php` at runtime)
 `/var/www/html/LocalSettings.php` | LocalSettings.php location, when passed in `/LocalSettings.php.template` will not be used. install.php & update.php will also not be run.
+
+### WikiBaseImport
```

Comment:
Rather than including docs for each extension we are going to add it would probably be better to link to the docs, but perhaps document how to interact with the maint scripts of extensions in a docker containers somewhere?